src/DotNetWorker.Core/Converters/Converter/SupportedConverterType.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Azure.Functions.Worker.Converters
+{
+    /// <summary>
+    /// An attribute that can specify a type of <see cref="Type"/> supported by function input conversion.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, AllowMultiple = true)]
+    public sealed class SupportedConverterTypeAttribute : Attribute
```

Comment:
Is this attribute used to specify the target types supported by the converter? Should the name be something like that? ex: InputConversionTargetType ?

Reply:
There is another attribute with similar name `InputConverterAttribute`. With the name `SupportedConverterTypeAttribute` we are trying to say that Converter supports this type. I am open to change it to a better name. Thoughts? cc @liliankasem @jviau

Reply:
I think we should leave it as it is. `InputConversionTargetType` could be confusing as it implies it's only for input and these supported type attributes can go on any converter